prettier.config.cjs

```diff
@@ -0,0 +1,28 @@
+/** @typedef  {import("prettier").Config} PrettierConfig */
+
+/** @type { PrettierConfig | SortImportsConfig } */
+const config = {
+  arrowParens: "always",
+  printWidth: 80,
+  singleQuote: false,
+  semi: true,
+  trailingComma: "all",
+  tabWidth: 2,
+  plugins: [
+    "@ianvs/prettier-plugin-sort-imports",
+    "prettier-plugin-tailwindcss",
+  ],
+  importOrder: [
```

Comment:
Can you add `"<TYPES>"` to the end of the sort order? Mixing type imports with regular imports gets a bit confusing at times.

Reply:
you want separate type imports? so like 

```ts
import { foo } from "foo";
import type { Bar } from "foo";
```

Reply:
Yes, exactly.